crew

```diff
@@ -334,28 +334,31 @@ def remove (pkgName)
     return
   end
 
-  Dir.chdir CREW_CONFIG_PATH do
-
-    #remove all files installed by the package
-    File.open("meta/#{pkgName}.filelist").each_line do |line|
-      begin
-        File.unlink line.chomp
-      rescue => exception #swallow exception
+  #if the package is marked as 'is_fake', the file/directorylists doesn't exist
```

Comment:
I'm talking about update not upgrade.  update changes `.rb` files without file/directory list modification.

1. test.rb is a regular package and there are file/directory lists in users environment
1. test.rb is modified to `is_fake` at the master
1. `crew update` changes all `.rb` files without file/directory lists modifications
1. when user type `crew upgrade test`, there still be file/directory lists but `test.rb` has `is_fake` mark

Last time I tried to make `is_fake` empty place holder few hours ago, it happens AFAIK.